filters/polarity/test.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
Could you add some longer examples and check how small changes will change the overall polarity?

Reply:
SImply, make sure that a small change gives a proportionally small polarity change in a long input.